supdupd.c

```diff
@@ -52,18 +62,18 @@
 #include <sgtty.h>
 #include <netdb.h>
 
-#ifdef TERMINFO
```

Comment:
Keep ifdef throughout the pull request.  Changing it to if defined() doesn't buy much.